ext/handlers/filters/message/message.go

```diff
@@ -47,12 +49,14 @@ func ForwardFromChatID(id int64) filters.Message {
 	}
 }
 
-func Regex(pattern string) filters.Message {
+func Regex(p string) filters.Message {
 	return func(m *gotgbot.Message) bool {
-		matched, err := regexp.MatchString(pattern, m.Text)
+		r, err := regexp.Compile(p)
 		if err != nil {
+			log.Fatal(err)
```

Comment:
Oh god
no please don't do that

Reply:
`log.Fatal(err)` 
this work is completely wrong
if we get an error, the application will completely terminated
it's not called error handling at all